scripts/UPDATE.sh

```diff
@@ -194,8 +195,10 @@ SuperBuild(){
         -DBUILD_siemens_to_ismrmrd=On \
         -DUSE_ITK=ON \
         -DDEVEL_BUILD=OFF\
+        -DNIFTYREG_USE_CUDA=OFF\
+        -DPYVER:STRING=3\
```

Comment:
```suggestion
```

Reply:
Why do you suggest to remove `-DPYVER=3`? Or you suggest to remove forcing it into the cache?

Reply:
`-DPYTHON_EXECUTABLE=/usr/bin/python3` should do the trick

Reply:
yes, I understood that after I posted! Thanks. I'm building a new VM from scratch as we speak.

Reply:
still, why is `-DPYTHON_EXECUTABLE` to be preferred to `-DPYVER-3`?
I am using that in the [github action](https://github.com/paskino/SIRF-SuperBuild/blob/build-github-action/.github/workflows/c-cpp.yml#L30) I made.

Reply:
in a controlled environment `PYVER=3` is fine, but once you have multiple python's, `PYTHON_EXECUTABLE` is far more reliable. It is also standard (while `PYVER` is our own work-around, and actually now obsolete)